index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <link rel="shortcut icon" href="./images/google.png" type="image/x-icon">
+    <title>Google</title>
+</head>
+<body>
+
+    <div class="userinfo">
+        <ul>
+            <li> <a href="#">Gmail</a></li>
+            <li> <a href="#">Images</a></li>
+            <li> 
+                <div class="tooltip">
+                    <img src="./images/appGoogle.png" alt="google app" class="googleapp"><span class="toolTipText"> Google apps</span>
+                </div>
+                </li>
+            <li> 
+                <div class="tooltipletter">
+                    <span class="letterborder">K</span> <span class="tooltipemail"><span class="ga"> Google Account</span> <br/> Kanu Mike Chibundu <br/> Michotall95@gmail.com</span>
+                </div>
+            </li>
+        </ul>
+    </div>
+
+    <div class="headGoogle">
+        <img src="./images/googleimg.png" alt="google image" class="googleimg">
+        <!-- <h1><span class="gcolor">G</span><span class="firsto">o</span><span class="secondo">o</span><span class="gcolor">g</span><span class="lcolor">l</span><span class="firsto">e</span></h1> -->
+    </div>
+
+    <div class="googlesearch">
+        <div class="imgsearch"> <img src="./images/search.png" alt="search icon"></div>
+        <form action="" class="formalign">
+            <input type="search" name="search" class="search">
+        </form>
+        <div class="imgmic"> <img src="./images/microphone.png" alt="microphone"></div>
+    </div>
+    
+    <div class="belowsearch more">
+        <h2>Google Search</h2>
+        <h2>!'m Feeling Lucky</h2>
+    </div>
+
+    <div class="belowsearch language">
+        <h3>Google offered in: <span class="lang">
+          <a href="#">Hausa</a>    
+          <a href="#">Igbo</a>   
+          <a href="#">Èdè Yorùbá</a>   
+          <a href="#">Nigerian Pidgin</a>  
+        </span></h3>
+    </div>
+
+    <footer>
+        <h3>Nigeria</h3><hr/>
+        <div class="footimg">
+            <img src="./images/orange.png" alt="orange image">
+            <h3>Carbon neutral since 2017</h2>
+        </div>
+
+        <div class="belowlink">
+
+            <section>
+                    <a href="#">About</a>
+                    <a href="#">Advertising</a>
+                    <a href="#">Business</a>
+                    <a href="#">How Search Works</a>
+            </section>
```

Comment:
- [ ] Why don't you use an `<ul>` tag instead of `<section>` and wrap `<a>` with `<li>`